bot/cogs/reddit.py

```diff
@@ -0,0 +1,310 @@
+from discord import Embed, Color
+from discord.ext.commands import Cog, Bot, Context, is_nsfw, group
+import random
+from bot import config
+import praw
```

Comment:
```suggestion
from praw import Reddit as RedditAPI
```
We're only using `Reddit` class from `praw` there is no point in importing the whole module.
I'm importing it as `RedditAPI` as `Reddit` name is already taken by the Cog class name, but feel free to use a different name
Note: you'll also need to replace every occurrence of `praw.Reddit` with just `Reddit`